acceptance_tests/features/edit_respondent_details.feature

```diff
@@ -43,46 +44,23 @@ Feature: As an internal user
 
   @us058-s001
   Scenario: The user is able to edit the email address of a respondent
-    Given the respondent with email "test_respondent@test.com" is enrolled
-    And the internal user has found the respondents details
+    Given the respondent with email "test_respondent2@test.com" is enrolled
+    And the internal user has found the second respondents details
```

Comment:
Rather than second respondent can we use their email or name as second respondent doesn't really mean anything.

Reply:
I'm going to use your suggestion below and use the email as a parameter :+1: